components/lib/password/Password.js

```diff
@@ -481,8 +481,18 @@ export const Password = React.memo(
         let input = <InputText {...inputTextProps} />;
 
         if (icon) {
+            const iconField = props.iconField;
```

Comment:
I don't think you need the `iconField` part?

Reply:
Well, I thought that in the future it would be easier to access the needed field value

Reply:
You can already change the icons with `props.hideIcon` etc.

```js
            if (unmaskedState) {
                icon = props.hideIcon || <EyeSlashIcon {...hideIconProps} />;
            } else {
                icon = props.showIcon || <EyeIcon {...showIconProps} />;
            }
```